Reference videos:
Try Catch: https://www.youtube.com/watch?v=iCFowjMYVd8&list=PLL34mf651faPB-LyEP0-a7Avp_RHO0Lsm&index=113

--------

1. Does this compile? If yes, what is the output? If no, fixed it
public static void main(String args[]) {
    try {
        int a = 30;
        int b = 0;
        int c = a / b;
        System.out.println ("Result = " + c);
    }
}

2. What is the output?
public static void main(String args[]) {
    try {
        System.out.println("try block");
        throw new IOException();
    } catch (FileNotFoundException e) {
        System.out.println("catch FileNotFoundException");
    } catch (IOException e) {
        System.out.println("catch IOException");
    } catch (Exception e) {
        System.out.println("catch Exception");
    } finally {
        System.out.println("finally block");
    }
}

3. If remove catch (IOException e) block from exercise 2, is there an error? If not, what is the output?